fast_and_reliable.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+"""intit the list and the set"""
+words_list = []
+words_set = set()
+file = open('words.txt')
```

Comment:
I think that you forgot to close the file at the end.
Close it or use with open instead.